### PR TITLE
Clean logicsig

### PR DESCRIPTION
**Pull request by ratik21**
- Removed
  -  `msgpack.encode`
  - `Rawlsig`
  - `hi-base32`
- Used sdk functions: `decodeAddress`, `logicSigFromByte`.
- Updated signatures in multisig examples because we updated account address in #138

### CODE REVIEW

File: examples/multisig/scripts/common/common.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ exports.executeTransaction = async function (deployer, txnParams){
     try {
        await executeTransaction(deployer, txnParams);
     } catch (e) {
-        console.error('Transaction Failed', e.response.error);
+        console.error('Transaction Failed', e.response ? e.response.error : e);
```
Comment (robert-zaremba) [inline examples/multisig/scripts/common/common.js:7]: I think you should be able to write:
```
        console.error('Transaction Failed', e.response.?error : e);
```

File: examples/multisig/scripts/multisig_sdk_sc.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ async function run(runtimeEnv, deployer) {
   //Get logic Signature
   const lsig = await deployer.loadLogic("sample-asc.teal", []);
 
-  lsig.sign(goldOwner.sk, mparams);  //lsig signed by gold-owner secret_key
+  lsig.sign(alice.sk, mparams);  //lsig signed by gold-owner secret_key
```
Comment (robert-zaremba) [inline examples/multisig/scripts/multisig_sdk_sc.js:22]: the comment has to be updated.

**Review by robert-zaremba — APPROVED** [review]
pre-approving (left 2 comments)

File: examples/multisig/scripts/multisig_sdk_sc.js (hunk 2 above)
Comment (ratik21) [reply examples/multisig/scripts/multisig_sdk_sc.js:22]: ok

File: examples/multisig/scripts/common/common.js (hunk 1 above)
Comment (ratik21) [reply examples/multisig/scripts/common/common.js:7]: It's showing an error: `Identifier expected`
image

**Pull request merged**